src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.container-card {
```

Comment:
`container-card` should be in this file, each BEM block should have separate scss file